export const captureException = (error: unknown) => {
  console.error(error);
};
